### ISSUE TITLE
Broad-based activation conflicts with stylelint

### ISSUE THREAD

**Issue by wycats** [ghmcadams/vscode-lintlens#65]
[This commit](4916cdf) removed support for configuring the location of eslintrc files in favor of broad-based detection.

Unfortunately, this detection activates for `.stylelintrc` files, which look like this:

```json
{
  "extends": ["stylelint-config-recommended-vue"],
  "files": ["**/*.postcss", "**/*.css"],
  "plugins": ["stylelint-gamut"],
  "rules": {
    "gamut/color-no-out-gamut-range": true,
    "color-function-notation": "modern",
    "at-rule-no-unknown": [
      true,
      {
        "ignoreAtRules": ["include"]
      }
    ]
  }
}
```

I'm not sure if it's even possible to differentiate between eslintrc files and stylelintrc files with structural detection.

What do you think?

**Comment by ghmcadams** [comment]
Hmmm... I haven't looked into it yet, but based solely on what you have pasted above, it looks like it won't be possible to tell them apart.  I will spend some time (when I have the time) to see what I have do.  Feel free to spend some time on this as well, if you would like a fix sooner than I can deliver.  Thanks for bringing this to my attention.  I appreciate it.

**Comment by ghmcadams** [comment]
@wycats One idea is to filter on the file name itself.  Though both tools allow for their config files to have any name, this will catch a vast majority of false positives (most people do not change the name of the config file).

**Comment by ghmcadams** [comment]
I implemented the above idea in the latest release.  I'm going to close this issue, given that it is sure to solve your issue (you mentioned the file name `.stylelintrc`)
